CONTRIBUTING.md

```diff
@@ -339,57 +357,6 @@ counter `L1-dcache-load-misses`
 
 TODO
 
-
-## Setting up continuous integration (CI) on your fork
```

Comment:
I think this passage is still valid, and should have a replacement.

I understand that it's less important, since Github Action is the main CI system now, but we should use this opportunity to educate our users (or future contributors, including from Meta).

What I would expect in this part : 
- A list of CI used to validate a PR.
- For each one : 
  - why they exist (what do they cover that other CI do not)
  - where is the corresponding test script (within the repository)
  - how to subscribe (link mostly, keep that on the provide side, since it tends to change over time)

I'm available for additional background information should you wish to.

Reply:
Force-pushed for the first and third bullet, will add second bullet shortly.